torch/_dynamo/variables/builtin.py

```diff
@@ -340,6 +373,15 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
+    def _call_reversible(self, tx, forward_name, reverse_name, args, kwargs):
+        # Call reverse (e.g. __radd__) if forward (e.g. __add__) isn't supported.
+        # This can be the case when doing int + SymInt, for example.
+        forward_res = args[0].call_method(tx, forward_name, args[1:], kwargs)
+        if forward_res is NotImplemented:
```

Comment:
If you see a NotImplemented here, you need to do the same probe for `args[1].call_method(tx, reverse_name, [args[0]], kwargs)` - and if that returns a NotImplemented, you should just invoke unimplemented.

This fn should return either a VariableBuilder, or raise unimplemented as @ezyang said.